### ISSUE TITLE
"Next" Button on Registration Page Triggers Cash Collection

### ISSUE THREAD

**Issue by franqq** [bespokeinteractive/registration#12]
Once a user clicks the next button. The system records the patients details and updates payment. Shoudn't cash collection be triggered by the print button on the next page? 
![payment print issue](https://cloud.githubusercontent.com/assets/5964987/11265504/bdc51846-8ead-11e5-8ab1-f888eaf4fc1b.PNG)

**Comment by bkirui** [comment]
@ndisha @franqq one of the proposed solution is to remove all functionality(Updating payment and patient details) from the next button to the print button on the following page, so that the clerk only prints receipt after receiving cash from the patient

**Comment by ndisha** [comment]
@bkirui I agree partly but I would suggest we have the 'Next' button save the patient details and the print receipt update payments while also moving the patient to the cashier queue. Would that work? 

The reason I say that is, if the patient does not have cash and we move all the saving functionality (for both patient details and payments) to the print button, what happens to the patient data if the cashier does not hit the print button? We loose it all?

**Comment by bkirui** [comment]
@ndisha your suggestion would also work. The implication will be that upon clicking the 'Next' button, patient details are saved including the Patient Category. The patient upon returning will be viewed as a revisit and also will have edit patient details to change category from let's say 'GENERAL' to 'WAIVER'. 

Looking at the earlier proposed solution, registration clerks usually collect cash before printing receipt. If we go this way it will be inline with how they normally operate

**Comment by ndisha** [comment]
@franqq @bkirui See the discussion here: https://docs.google.com/document/d/1VJiKYb4_xiy8auKkMLvnv8ogCKs3OYIl19_9AT8_6YQ/edit
